libs/blog/partners/ui-partners/src/lib/partners/partners.component.html

```diff
@@ -1,11 +1,19 @@
 <section
   aria-labelledby="partners-title"
-  class="bg-al-card my-16 flex w-full flex-col justify-between rounded-lg lg:mb-32 lg:mt-16"
+  class="bg-al-card lg:mt-16' my-16 flex w-full flex-col justify-between rounded-lg lg:mb-32"
 >
-  <div class="align self-start py-8 pl-8">
+  <div class="py-8 pl-8">
     <h2 id="partners-title" *transloco="let t" class="text-2xl font-bold">
-      {{ t('homePage.partners') }}
+      {{
+        partnerType() === 'main-partner'
+          ? t('homePage.mainPartner')
+          : t('homePage.partners')
+      }}
     </h2>
   </div>
-  <al-partners-list></al-partners-list>
+  @if (partnerType() === 'main-partner') {
+    <al-main-partner></al-main-partner>
+  } @else {
+    <al-partners-list></al-partners-list>
+  }
```

Comment:
Instead of creating new components, pass title and image to display in inputs

Reply:
I can not agree. Look at this (main partner component)
`  protected readonly hoaIcon = computed(() => {
    if (this.lang() === 'en') return this._hoaIconHireUs;
    else return this._hoaIconHiring;
  });`
  As you can see, it depends on the chosen language. The logic in these two components is quite different. We do not need to inject `translocoService` in `partners-list.component`. Let me know what do you think about it